core/src/main/java/org/mskcc/cbio/portal/util/DebugHrrsFilter.java

```diff
@@ -50,8 +51,10 @@
 
 public class DebugHrrsFilter extends Base64HrrsFilter {
 
+    private static final String DEFAULT_HRRS_LOGGING_NAME = "/hrrs-logging.csv";
+
     private static RotationConfig createRotationConfig() {
```

Comment:
@averyniceday 	is it possible to not run `super(createRotationConfig())` if `hrrs.enable.logging` is false? I assume `java.io.tmpdir` is always set but seems redundant to set up a logging file path if the whole feature is disabled?

Reply:
@inodb I'll look into it some more. But the superclass Base64HrrsFilter requires a RotationConfig argument passed in.

Reply:
@averyniceday 	I see - not worth spending too much time on it, but if it's an easy fix that would be great 👍 - gonna approve for now

Reply:
@inodb I changed it so this class will extend the HrrsFilter which doesn't require a RotationConfig passed in to the constructor. Then the writer/writerTarget is only set if the logging is enabled. I left the code for defaulting to the tempdir just in case someone enables logging with no path.

Reply:
@averyniceday perfect!